scripts/import_gff3.py

```diff
@@ -0,0 +1,347 @@
+        # Eliminating ;
+        for i in attr_names:
+            atr_data[i] = atr_data[i].replace({';': ''}, regex=True)
+    
+        # desglosing the expression column in a column for each sample
+        list_expression=[]
+        for row in range(atr_data.shape[0]):
+            list_expression.append(atr_data.values[row,expression_colindex].split(','))
+        sample_names=['Expression_' +x for x in sample_names]
+        
+        expression_data = pd.DataFrame(list_expression, columns=sample_names)
+        
+        atr_data=atr_data.drop('Expression',axis=1) #Remove the expression column
+        atr_data=atr_data.join(expression_data)
+    
+        # Joining the body and attributes dataframes
+        data = body_data.join(atr_data)
+    
+        return data
+    except:
+        print 'Error loading the file'
+
+
+"""
+Function that check the header then load a gff file and check the content
+Returns the dataframe if the format is ok, false if not
+"""
+def load_gff3(filename):
+    try:
+        Error = False     
+        # Checking the format file 
+        # Header and 1st data row
+        header_rows=0
+        with open(filename) as file:
+            rowfile=file.readline()
+            header_rows=header_rows+1
+
+            while True:
+                if rowfile.startswith('#'):
+                    rowfile=file.readline()
+                    header_rows=header_rows+1
+                else:
+                    data_1=rowfile.split('\t')
+
+                    if header_rows!=4:
+                        Error = True
+                        print 'Bad header' 
+                    break
+        #Number of columns without breaking down attributes column 
+        if len(data_1) > 9:
+            Error=True
+            print(len(data_1))
+            print('Too much columns')
+        # Cheking the attributes column  
+        attr_names=data_1[-1].split(';')
+        list_attr=[]
+        for atr in range(len(attr_names)-1):
+            list_attr.append(attr_names[atr].split()[0])
+        possible_attr = ['UID', 'Read', 'Name', 'Parent', 'Variant', 'Cigar', 'Hits', 'Alias', 'Genomic', 'Expression',
+                           'Filter', 'Seed_fam']
+        for attr in list_attr:    
+            if attr not in possible_attr:
+                Error=True
+                print attr,'is not a possible attribute'
+                break  
+        if Error:
+            print 'File format error'            
+            return False      
+        # If not format error, loading content
+        try:
+            dataframe=loadfile(filename,True)
+        except:
+            print 'Error loading file'
+            return False
+        print 'Checking content'
+        for i in range(dataframe.shape[0]):
+            # Labels in type column
+            if dataframe.loc[i, 'type'] not in ['ref_miRNA', 'isomiR']:
+                Error = True
+                print('line', i, 'bad type error')
```

Comment:
I think python 2.7 should be print character_string, like you were using before. This seems python 3* or importing _future library, right?

Reply:
Corrected.